skia-bindings/build_support/skia.rs

```diff
@@ -692,7 +700,7 @@ pub fn build_skia(
     ninja_command: &Path,
 ) {
     let ninja_status = Command::new(ninja_command)
-        .args(&["-C", config.output_directory.to_str().unwrap()])
+        .args(&[":skia", "-C", config.output_directory.to_str().unwrap()])
```

Comment:
Note: I did this so that it wouldn't build other executable (example) targets, and only build `libskia.a`. This avoids some other linker issues.